src/backend/distributed/commands/database.c

```diff
@@ -256,6 +284,34 @@ PreprocessAlterDatabaseRefreshCollStmt(Node *node, const char *queryString,
 
 #endif
 
+/*
+ * PreprocessAlterDatabaseRenameStmt is executed before the statement is applied to the local
+ * postgres instance. In this stage we prepare ALTER DATABASE RENAME statement to be run on
+ * all workers.
+ */
+List *
+PreprocessAlterDatabaseRenameStmt(Node *node, const char *queryString,
```

Comment:
Let's use Postprocess for that as we agreed on before, unless preprocess doesn't allow a particular check / break something ..

Reply:
Fixed